agimus_controller_ros/launch/agimus_controller.launch.py

```diff
@@ -0,0 +1,27 @@
+from launch import LaunchDescription
+from launch.substitutions import PathJoinSubstitution
+
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+
+
+def generate_launch_description():
+    mpc_params = PathJoinSubstitution(
+        [
+            FindPackageShare("agimus_demo_01_lfc_alone"),
+            "config",
+            "linear_feedback_controller.yaml",
+        ]
+    )
+    print("parameters of the mpc", mpc_params)
```

Comment:
This will print `PathJoinSubstitution` object pointer instance and not the string. Plus prints are not very usable in ROS 2 launch files